💄 [UI/UX] Restructure trek detail UI (refs #2967) (#4422)
GeotrekCE/Geotrek-admin: docs/changelog.rst

2.111.0+dev     (XXXX-XX-XX)
----------------------------

**UI/UX**

> - Move an object's related objects from the properties tab into their own tab

Reviewer: ```suggestion
- Move the related objects from the properties tab into their own tab, on objects details pages
```

> - Move an object's related objects from the properties tab into their own tab (refs #2967)

Reviewer: ```suggestion
- Move the related objects from the properties tab into their own tab, on objects details pages
```
